docs/compression_algorithms/Quantization.md

```diff
@@ -109,6 +109,15 @@ The quantization configuration in the `"target_device": "NONE"` case may be over
 
 For all target HW types, parts of the model graph can be marked as non-quantizable by using the `"ignored_scopes"` field - inputs and weights of matching nodes in the NNCF internal graph representation will not be quantized, and the downstream quantizers will not propagate upwards through such nodes.
 
+---
+**NOTE**
+
+There is a known issue with AVX2 and AVX512 CPU devices. The issue appears with 8-bit matrix calculations with tensors that are close to the maximum or saturated value.
```

Comment:
More info needed (specify that the issue affects accuracy in the INT8 runtime).

Reply:
Please take a look at the updated version